CMakeLists.txt

```diff
@@ -155,6 +154,9 @@ option(NGRAPH_ARTIFACTS_DIR "Where would nGraph be installed after build" FALSE)
 option(UNIT_TEST_ENABLE "Control the building of unit tests" FALSE)
 option(UNIT_TEST_TF_CC_DIR "Location where TensorFlow CC library is located" FALSE)
 option(NGRAPH_BRIDGE_STATIC_LIB_ENABLE "Enable build nGraph bridge static library" FALSE)
+option(ENABLE_OPENVINO "Build OpenVINO backend" FALSE)
+option(USE_PREBUILT_OPENVINO "Use prebuilt OpenVINO located in OPENVINO_ARTIFACTS_DIR" FALSE)
+option(OPENVINO_ARTIFACTS_DIR "Where would OpenVINO be installed after build" FALSE)
```

Comment:
Good point. I don't think cmake's options are typed; if no initial value is provided it uses "OFF". I simply followed the convention we use for other variables above because we mostly seem to be checking for the presence of a value by doing `if(<OPTION_NAME>)`.